### DISCUSSION TITLE
Touchpad becomes invisible to Windows after saving changes in UMAF

### DISCUSSION THREAD

**Discussion by kykym** [DavidS95/Smokeless_UMAF#23] [Q&A]
Hello everyone, and thank you for this tool

I'm using an HP omen laptop, r5 5600h and rx6600m 

I'm interested in using the UMAF to disable some settings like SmartShift and change power values.

However, every time I make changes in the UMAF and save them, Windows starts not to see my touchpad. The system states that there's simply no such thing connected

HP UEFI sees touchpad just fine

I tried to reapply some settings concerning touchpad in the UMAF but with zero results

Problem can be solved only through CMOS reset

**Comment by DavidS95** [comment]
Any change does this? That is a weird issue. UMAF does not have trackpad or mouse support but after applying the settings going to windows shows nothing? Have u tried doing these changes then going to HP bios then doing more changes and seeing if it fixes ur trackpad.

**Reply by kykym** [reply]
I did some tests today and discovered it was cool when I deactivated and saved some settings in UMAF.
But the problem arose when I implemented the default settings in UMAF.
Nothing in the standard BIOS helps.

**Reply by Shvarsman** [reply]
I have the same problem. Nothing helps even resetting CMOS. How can I get the touchpad back?

**Comment by Espionage724** [comment]
I'd boot a Linux install and reboot twice from it with `reboot=pci` to cause a hard firmware reboot (might be easy with Linux LiveUSB but afaik the reboot option only applies on next _persisted_ reboot?)

Maybe also reboot that Linux install with different options like `acpi_osi=Windows 2013` to cause BIOS to initialize devices differently.
